crates/ruff_python_parser/tests/snapshots/invalid_syntax@expressions__if__recover.py.snap

```diff
@@ -309,7 +309,7 @@ Module(
 1 | # Invalid test expression
 2 | x if *expr else y
 3 | x if lambda x: x else y
-  |      ^^^^^^^^^^^ Syntax Error: unparenthesized lambda expression cannot be used here
+  |      ^^^^^^^^^^^ Syntax Error: `lambda` expression cannot be used here
```

Comment:
You quote `lambda` in this error message but `yield` isn't quoted in the error message below. I think we should either always or never quote the expression name.

Reply:
There are other inconsistencies with error messages, I'll club this with them in a single PR. Thanks for noticing.